src/lib/coda_lib/coda_lib.ml

```diff
@@ -164,24 +164,25 @@ let sync_status t =
         match online_status with
         | `Offline ->
             if `Empty = first_connection then (
-              Logger.info (Logger.create ()) ~module_:__MODULE__
-                ~location:__LOC__ "Coda daemon is now connecting" ;
+              Logger.info t.config.logger ~module_:__MODULE__ ~location:__LOC__
+                "Coda daemon is now connecting" ;
               `Connecting )
             else if `Empty = first_message then (
-              Logger.info (Logger.create ()) ~module_:__MODULE__
-                ~location:__LOC__ "Coda daemon is now listening" ;
+              Logger.info t.config.logger ~module_:__MODULE__ ~location:__LOC__
+                "Coda daemon is now listening" ;
```

Comment:
these need to be debounced too

Reply:
(evidence: if you spam coda status before bootstrap starts, these spam in the same way)

Reply:
This PR doesn't debounce at all. I can do that in this PR I think but I'd rather leave it to @wu-s-john. If these lines are logged, it should be because that's the true status of the daemon- even if it's already told you that twice already.